modin/pandas/test/utils.py

```diff
@@ -868,12 +868,43 @@ def generate_none_dfs():
     return df, df2
 
 
+def make_dict_hash(dict_to_hash):
```

Comment:
Why not just using:

```python
import uuid
unique_id = uuid.uuid1().hex
```

Reply:
This approach is better than getting hex from kwargs dict, so it was replaced with your suggestion.